src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,22 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Can't login user", e);
             return;
         }
         OrderService orderService = new OrderServiceImpl();
-        orderService.completeOrder(user.getUserId());
+        orderService.completeOrder(123L);
```

Comment:
I suggest not touching logic that was here.
```suggestion
        orderService.completeOrder(user.getUserId());
```